src/Type/PostObject/Mutation/PostObjectMutation.php

```diff
@@ -89,16 +69,53 @@ public static function input_fields( $post_type_object ) {
 					'type'        => Types::post_status_enum(),
 					'description' => __( 'The status of the object', 'wp-graphql' ),
 				],
-				'title'         => [
-					'type'        => Types::string(),
-					'description' => __( 'The title of the post', 'wp-graphql' ),
-				],
 				'toPing'        => [
 					'type'        => Types::list_of( Types::string() ),
 					'description' => __( 'URLs queued to be pinged.', 'wp-graphql' ),
 				],
 			];
 
+			if ( post_type_supports( $post_type_object->name, 'author' ) ) {
```

Comment:
i might suggest to keep the definitions in the array above and `unset` them here if they aren't supported. this keeps all the definitions together and in alpha order for readability. just a mild preference though

Reply:
@chriszarate I like that idea